www/js/con-author.js

```diff
@@ -355,20 +354,30 @@ define([
 			*/
 		},
 
-		handleDynamic: function(dynamic){
-			// Summary: Sets the current node to be parent node
-			console.log("********************* in handleDynamic", dynamic);
-			this._model.authored.setAccumulator(this.currentID, dynamic);
-			/*TODO : Add code to update position of the node*/
-			/*
-			this.logging.log("solution-step", {
-				type: "solution-enter",
-				nodeID: this.currentID,
-				property: "dynamic",
-				node: this._model.given.getName(this.currentID),
-				value: dynamic
-			});
-			*/
+		handleVariableType: function(event){
```

Comment:
I had used event.target.value in handler to get the value of the checked radio button.